calibre-web-automated-downloader/src/backend.rs

```diff
@@ -0,0 +1,136 @@
+use crate::book_manager;
+use crate::config::CONFIG;
+use crate::models::{BookInfo, QueueStatus, BOOK_QUEUE};
+use lazy_static::lazy_static;
+use log::{error, info};
+use serde_json::json;
+use std::collections::HashMap;
+use std::{fs::File, io::Read, path::Path};
+use tokio::sync::Mutex;
+
+lazy_static! {
```

Comment:
Don't we already have a mechanism for this for `CONFIG` global static variable? Wh should we add another library for this task?